google/cloud/storage/examples/storage_bucket_samples.cc

```diff
@@ -55,20 +55,24 @@ void ListBuckets(google::cloud::storage::Client client, int& argc,
   [](gcs::Client client) {
     int count = 0;
     StatusOr<gcs::ListBucketsReader> bucket_list = client.ListBuckets();
+
     if (!bucket_list) {
         std::cerr << "Error reading bucket list for default project"
                   << ", status=" << bucket_list.status() << std::endl;
         return;
     }
-    for (auto&& meta : *bucket_list) {
-      if (!meta.ok()) {
+
+    for (auto&& bucket_metadata : *bucket_list) {
+      if (!bucket_metadata.ok()) {
```

Comment:
Following what the other changes have, `.ok()` needs to be removed. 

https://github.com/googleapis/google-cloud-cpp/blob/695675ce6c0ee169786e26388f4852032ba23235/google/cloud/storage/examples/storage_bucket_samples.cc#L66

https://github.com/googleapis/google-cloud-cpp/blob/695675ce6c0ee169786e26388f4852032ba23235/google/cloud/storage/examples/storage_bucket_samples.cc#L95

https://github.com/googleapis/google-cloud-cpp/blob/695675ce6c0ee169786e26388f4852032ba23235/google/cloud/storage/examples/storage_bucket_samples.cc#L204

https://github.com/googleapis/google-cloud-cpp/blob/695675ce6c0ee169786e26388f4852032ba23235/google/cloud/storage/examples/storage_bucket_samples.cc#L544

https://github.com/googleapis/google-cloud-cpp/blob/695675ce6c0ee169786e26388f4852032ba23235/google/cloud/storage/examples/storage_bucket_samples.cc#L862